src/Agent.Worker/Handlers/ProcessHandler.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Microsoft.TeamFoundation.DistributedTask.WebApi;
+using Microsoft.VisualStudio.Services.Agent.Util;
+using Microsoft.VisualStudio.Services.Agent.Worker;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Microsoft.VisualStudio.Services.Agent.Worker.Handlers
+{
+    [ServiceLocator(Default = typeof(ProcessHandler))]
+    public interface IProcessHandler : IHandler
+    {
+        ProcessHandlerData Data { get; set; }
+    }
+
+    public sealed class ProcessHandler : Handler, IProcessHandler
```

Comment:
As an example I wanted to add one more handler so the pattern is clear. The process handler isn't fully implemented.